client/blocks/conversations/list.jsx

```diff
@@ -123,12 +124,14 @@ export class ConversationCommentList extends React.Component {
 			commentsTree,
 			Object.keys( this.getCommentsToShow() )
 		);
-		inaccessible.forEach( commentId => {
-			nextProps.requestComment( {
-				commentId,
-				siteId,
+		inaccessible
+			.filter( commentId => ! commentErrors[ `${ siteId }-${ commentId }` ] )
```

Comment:
Should we have a key maker for this perhaps, in `state/comments/utils`?

Reply:
this is the only current usage of the key so I don't feel a need to yet.
i do want to make a better place for us to put all our keyMakers though.

something like:
```js
// reader/keymakers.js
export commentErrors = ...;
export siteItems = ...;


// consumer.js
import * as  keyMakers from 'reader/keymakers';

keyMakers.commentError( siteId, commentId );
```

Reply:
> this is the only current usage of the key

It's repeated a couple of times in the reducer.js :)

`const key = `${ action.siteId }-${ action.commentId }`;`

All the other comments reducers use `getStateKey` to create their key, but since the key here is different, I think there's a case for having `getErrorKey` or similar.

Reply:
yea, they ideally wouldn't be named getStateKey since that is too generic.
what about a `binaryKey(arg1, arg2)`

Reply:
im convinced